pkg/place/operations/find.go

```diff
@@ -44,21 +44,28 @@ func Find(ctx context.Context, backendNames []string, pattern string, force bool
 	instances := new(ConcurrentSlice)
 
 	for _, name := range backendNames {
+		bucketName := name
```

Comment:
`backendName`

Reply:
no this is not mistake this is name of the bucket in cache